tests/test_client.py

```diff
@@ -309,6 +309,9 @@ async def test_dispatch_stream(client: FakeClient, sample: Dispatch) -> None:
 
     stream = client.stream(microgrid_id)
 
+    # Call function again to test behavior if stream already exists
+    _ = client.stream(microgrid_id)
+
```

Comment:
Wouldn't it make sense to create a separate test for the double call? What if there is a bug that affects the `stream()` function in a way that if doesn't work when called only once but works when called twice?